src/client/components/features/Stages3eme/Rechercher/FormulaireRecherche/FormulaireRechercheStages3eme.test.tsx

```diff
@@ -11,45 +11,157 @@ import {
 import { mockUseRouter } from '~/client/components/useRouter.mock';
 import { mockSmallScreen } from '~/client/components/window.mock';
 import { DependenciesProvider } from '~/client/context/dependenciesContainer.context';
+import { aCommuneQuery } from '~/client/hooks/useCommuneQuery';
+import { aLocalisationService } from '~/client/services/localisation/localisation.service.fixture';
 import { aMetierService } from '~/client/services/metiers/metier.fixture';
 import { aStage3emeService } from '~/client/services/stage3eme/stage3eme.service.fixture';
 import { createSuccess } from '~/server/errors/either';
+import {
+	aCommune,
+	aRésultatsRechercheCommune,
+} from '~/server/localisations/domain/localisationAvecCoordonnées.fixture';
 
 describe('FormulaireRechercheStages3eme', () => {
 	beforeEach(() => {
 		mockSmallScreen();
 	});
-	describe('quand on recherche par métier', () => {
-		it('ajoute le métier recherché aux query params', async () => {
-			// GIVEN
-			const routerPush = jest.fn();
-			const user = userEvent.setup();
-			mockUseRouter({ push: routerPush });
-			const metierService = aMetierService();
-			jest.spyOn(metierService, 'rechercherMetier').mockResolvedValue(createSuccess([
-				{ code: 'codeMetier', label: 'boulanger' },
-			]));
-
-			render(
-				<DependenciesProvider
-					stage3emeService={aStage3emeService()}
-					metierStage3emeService={metierService}
-				>
-					<FormulaireRechercheStages3eme />
-				</DependenciesProvider>,
-			);
-
-			// WHEN
-			const inputRechercheMetier = screen.getByRole('combobox', { name: 'Métier (facultatif)' });
-			await user.type(inputRechercheMetier, 'boulanger');
-			const boulangerOption = await screen.findByRole('option', { name: 'boulanger' });
-			await user.click(boulangerOption);
-			const buttonRechercher = screen.getByRole('button', { name: 'Rechercher' });
-			await user.click(buttonRechercher);
-
-			// THEN
-			// NOTE (DORO 22-11-2023): Query params "location=here" temporaire pour afficher les résultats de recherche (à remplacer par les vrais query params quand la recherche par localisation sera implémentée)
-			expect(routerPush).toHaveBeenCalledWith({ query: 'location=here&libelleMetier=boulanger&codeMetier=codeMetier' }, undefined, { shallow: true });
+
+	it('quand la query comporte la localisation, ajoute la valeur par default dans l‘input', () => {
```

Comment:
```suggestion
	it('quand la query comporte la localisation, ajoute la valeur correspondante dans l‘input', () => {
```
nitpick mais le "par défaut" m'a un peu fait bugué